seed_data/utils_test.py

```diff
@@ -0,0 +1,25 @@
+"""Tests for utils file"""
+import datetime
+import pytz
+from django.test import TestCase
+
+from seed_data.utils import add_year
+
+
+class AddYearTests(TestCase):
+    """Tests for add year method"""
+
+    def test_add_year(self):  # pylint: disable=no-self-use
+        """assert add year method works"""
+        now = datetime.datetime.now(tz=pytz.UTC)
```

Comment:
@amir-qayyum-khan not sure if you missed this, but this test will definitely fail down the road. Particularly on the next leap day `day = 29`, but there won't be a 29th on the next year. I think you need to remove this test and assert specific behaviors around specific dates in and around the leap day. Particularly:

- Feb 28, 2015
- Feb 28, 2016
- Feb 29, 2016